draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2837,7 +2877,7 @@ uCDN-A                         dCDN-B                         dCDN-C
       <vspace blankLines="1"/>
       Value: A trigger state, as defined in <xref target="trigger-state"/>.
       <vspace blankLines="1"/>
-      Mandatory: One of "state" or "label" should be present.
+      Mandatory: One of the "state" or "label" should be present.
```

Comment:
Adding "the" sounds more awkward here. We're referring to one of these choices. Maybe "either" is is a better choice than "the".